test/higher_order_ops/test_local_map.py

```diff
@@ -653,6 +651,42 @@ def forward(self, x):
             )
         ap_style_initial_capture(model, inputs)
 
+    @unittest.skipIf(*get_skip_reasons())
+    def test_none_placements(self):
+        class ScalarHolder(torch.nn.Module):
+            def __init__(self, scalar):
+                super().__init__()
+                self.scalar = scalar
+
+            def forward(self, x):
+                return x + self.scalar
+
+        @local_map(
+            out_placements=((Replicate(), Replicate(), Replicate()),),
+            in_placements=(
+                (Replicate(), Replicate(), Replicate()),
+                None,
+                None,
```

Comment:
do you mind explaining the intended usage pattern? (how do the in_placement arguments here map to the function signature)

(1) `in_placements` here is a nested tuple (first arg is tuple of 3 Replicates, and args 2 and 3 are `None`)

(2) the fn itself accepts a tensor, scalar, and module, where I imagine dynamo will desugar the module into it's internal state (which is a single scalar).

So I don't fully follow how the nested structure of in_placements (5 total args) maps to the function signature (4 args), but I'm probably missing something.